src/main/jbake/content/blog/2017/02/03/grafana-events.adoc

```diff
@@ -7,6 +7,14 @@ Joel Takvorian
 
 The latest release of the link:https://github.com/hawkular/hawkular-grafana-datasource/releases/tag/v1.0.7[Hawkular Grafana plugin] (1.0.7) introduced a new feature, implementing link:http://docs.grafana.org/reference/annotations/[Grafana Annotations].
 
+ifndef::env-github[]
+image::/img/blog/2017/2017-02-03-annotations-tooltip-2.png[Logged error]
```

Comment:
I think it would be good to center the image
See http://asciidoctor.org/docs/user-manual/#positioning-attributes